libs/dh/shared/feature-authorization/src/lib/dh-inactivity-logout.component.ts

```diff
@@ -36,18 +36,21 @@ import { WattTypedModal, WATT_MODAL } from '@energinet-datahub/watt/modal';
   template: `
     <watt-modal *transloco="let t" [size]="'small'" [title]="t('sessionExpirationTitle')">
       <p>{{ t('sessionExpirationContentPartA') }}<br />{{ t('sessionExpirationContentPartB') }}</p>
-      <h2>{{ warningCountdown$ | push | date: 'mm:ss' }}</h2>
+      <h2>{{ warningCountdown() | date: 'mm:ss' }}</h2>
     </watt-modal>
   `,
-  imports: [RxPush, DatePipe, TranslocoDirective, WATT_MODAL],
+  imports: [DatePipe, TranslocoDirective, WATT_MODAL],
 })
 export class DhInactivityLogoutComponent extends WattTypedModal {
   private readonly secondsUntilLogOff = 10;
 
-  readonly warningCountdown$ = timer(0, 1000).pipe(
-    take(this.secondsUntilLogOff + 1),
-    tap((elapsed) => elapsed >= this.secondsUntilLogOff && this.dialogRef.close(true)),
-    map((elapsed) => Math.max(0, this.secondsUntilLogOff - elapsed - 1)),
-    map((elapsed) => new Date(elapsed * 1000))
+  readonly warningCountdown = toSignal(
+    timer(0, 1000).pipe(
+      take(this.secondsUntilLogOff + 1),
+      tap((elapsed) => elapsed >= this.secondsUntilLogOff && this.dialogRef.close(true)),
+      map((elapsed) => Math.max(0, this.secondsUntilLogOff - elapsed - 1)),
+      map((elapsed) => new Date(elapsed * 1000))
+    ),
+    { initialValue: new Date(this.secondsUntilLogOff * 1000) }
```

Comment:
suggestion: use `requireSync` instead, since `timer(0, 1000)` should emit synchronously.